documentation/en/default-lotus-miner-config.toml

```diff
@@ -466,6 +468,22 @@
   # env var: LOTUS_PROVING_SINGLERECOVERINGPARTITIONPERPOSTMESSAGE
   #SingleRecoveringPartitionPerPostMessage = false
 
+  # Maximum number of partition batches to process in parellel. 0 = no limit.
+  # 
+  # A partition batch after being processed ends up as a PoSt message, so this limit can also be thought of as "how
+  # many WindowPoSt messages to create in parallel".
```

Comment:
Little confused here - I thought we were processing multiple partitions in the same WdPost msg in parallel

Reply:
No, it's exactly as this comment says, the loop in `runPoStCycle` starts a goroutine per batch, and a batch can contain more than one partition. We're not able to compute partitions within one batch in parallel due to how sector skipping / challenge generation used to work in proofs. The new sector challenge generation has now landed in the network, but we still didn't update the windowpost logic to do the simpler skipping (and that's not something I want to do as part of this PR)